pkg/api/v1/cluster_validation.go

```diff
@@ -42,9 +41,9 @@ func checkValues(c *ScyllaCluster) error {
 	}
 
 	if len(c.Spec.ScyllaArgs) > 0 {
-		version, err := semver.Parse(c.Spec.Version)
-		if err == nil && version.LT(ScyllaVersionThatSupportsArgs) {
-			return errors.Errorf("ScyllaArgs is only supported starting from %s", ScyllaVersionThatSupportsArgsText)
+		version := NewScyllaVersion(c.Spec.Version)
+		if version.SupportFeatureUnsafe(ScyllaVersionThatSupportsArgs) {
+			return errors.Errorf("ScyllaArgs is only supported starting from %s", ScyllaVersionThatSupportsArgs)
```

Comment:
This is a regression. 
```suggestion
		version := NewScyllaVersion(c.Spec.Version)
		if !version.SupportFeatureUnsafe(ScyllaVersionThatSupportsArgs) {
			return errors.Errorf("ScyllaArgs is only supported starting from %s", ScyllaVersionThatSupportsArgs)
```

Reply:
Should be good now

Reply:
It is still the same, perhaps you forgot to amend your commit :)

Reply:
Ah I modified the one in config.go 😅 . Thanks!